all/pom.xml

```diff
@@ -340,14 +445,6 @@
       <scope>compile</scope>
       <optional>true</optional>
```

Comment:
actually I think we should remove epoll here as well. We should only pull this into the all if -Puber is used.